### ISSUE TITLE
Camera Path >"add from camera" not working. Orientation and placement messed up

### ISSUE THREAD

**Issue by TheLordSaiyan** [NVlabs/instant-ngp#1286]
Hello everyone,
Everything works perfectly and it's amazing, until i try to make a video.
When I click on "Add from camera", it creates a camera but not the view I wanted. And when i zoom out it seems the world orientation doesnt match with the one of the crop box.. any idea how to solve this ? Have I miss something ?
thank you for your time

**What I Want**
(https://user-images.githubusercontent.com/127543061/224394044-0b8abff4-23e6-4880-afdd-4f98bc3dc94f.jpg)
**What I get** 
(https://user-images.githubusercontent.com/127543061/224394086-b2fe5625-8232-49e2-bf3b-b691d694f2c7.jpg)
**Zoomed out**
(https://user-images.githubusercontent.com/127543061/224394093-0528ea00-0bcb-4763-bbe0-cf6e585c0121.jpg)

**Comment by Tom94** [comment]
Have you tried updating to the latest version of the code / binaries?

**Comment by TheLordSaiyan** [comment]
I think I have, but let's imagine I haven't. How do I update it ? 
P.S.: I'm not a developer/coder , so be nice with me, if I sound like a noob/5 years old who doesnt know what he is doing.
@Tom94 Thank you for your quick answer

**Comment by bySnach** [comment]
I have exactly the same issue as @TheLordSaiyan described above (with i**nstant-ngp**).
I've been trying to solve this problem myself for a long time already, but without any success.

The only workaround I found is using - [nerfstudio](https://docs.nerf.studio/en/latest/) instead (but only as an alternative method). The camera path is working stably there.

**Comment by TheLordSaiyan** [comment]
@bySnach I did it.. I'm finally able to add a camera and export my video in HD (not 4K unfortunatelly)
What I did :
- Deleted the whole instant-ngp file
- Be sure to have everything else installed (Python, Visual Studio 2019, CMake and CUDA - I this order exactly) Refer the repo for which versions specifically MAKE SURE TO ADD TO THE PATH OTHERWISE NOTHING WILL WORK -- I recommend downloading Optix and Vulkan SDK also
- Open your favorite command propter, Download/Clone the repo (choose one)
cd **[Direction where your want it to be installed]** 
pip install https://github.com/NVlabs/instant-ngp.git

cd **[Direction where your want it to be installed]** 
git clone https://github.com/NVlabs/instant-ngp.git
- Then install all the requirement needed using the command
pip install -r requirements.txt
- Choose the correct release for your corresponding card and extract it in the same path as the main repo, and change the cd on your command promp to be on the folder you just downloaded
- run pip install -r requirements.txt
- Then install the externals given (COLMAP and FFMPEG) - I also ran these commands, just to be sure
**COLMAP**
pip install https://github.com/colmap/pycolmap.git
pip install pycolmap
FFMPEG
pip install ffmpeg-python
git clone https://git.ffmpeg.org/ffmpeg.git ffmpeg
-Make sure to carrefully read the README.txt in every subfolder, because you might be missing some scripts for everything to run smoothly, and run the command pip install -r requirements.txt on every Requirements you see on the subfolders just to be sure.
If any of this may seem wrong or I incorectly used a word, I'm sorry, this is not my field of expertise.
That's how I fixed my problem, hope it will do it for you do. Be blessed.
